### PR TITLE
Update run_translation.py broken link example Pytoch

### LINKED ISSUE #25444: Access to Transformers example Pytorch link broken . Impact on navigation as well
Hello there! 👋

## Context
Exploring Pytorch translation example

## Issue
There seems to be a broken link in Pytorch example in the Translation example, similar to what happened in #24497 
https://github.com/huggingface/transformers/blob/347001237a8ff845fc23f678107fc505361f9f13/examples/pytorch/translation/run_translation.py#L381

When I follow the link , the following message appears. And when I click the here link

Captura de pantalla 2023-08-10 a las 18 19 10

It redirects me to https://huggingface.co/docs/transformers/main/en/examples with an 404 error.

## Potential fix 

1. Is that ok if I submit a quick fix as in #24594 ?
2. Would it be hard to try to submit a contrib of something like this? ... 
> specify doc redirects using a `redirects.yml` like in datasets and other libraries: https://github.com/huggingface/datasets/blob/main/docs/source/_redirects.yml

Do you know if this entails more than create than source folder and .yaml file? Would I have to study [this](https://github.com/huggingface/datasets/tree/main/docs#generating-the-documentation)?

@sgugger 
Keep up the good work!

### PR DESCRIPTION
**Pull request by SoyGema**
# What does this PR do?

Fixes #25444 

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [X] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@sgugger

### CODE REVIEW

File: examples/pytorch/translation/run_translation.py [hunk 1]
```diff
@@ -378,7 +378,7 @@ def main():
             token=model_args.token,
         )
     # See more about loading any type of standard or custom dataset (from files, python dict, pandas DataFrame, etc) at
-    # https://huggingface.co/docs/datasets/loading_datasets.html.
+    # https://huggingface.co/docs/datasets/v2.13.1/loading.
```
Comment (sgugger) [inline examples/pytorch/translation/run_translation.py:381]: Would be nice to not ping a specific version here, as the link will be quickly outdated.

**Review by sgugger — COMMENTED** [review]
Thanks for your PR! Left a small comment.

**Review by sgugger — APPROVED** [review]

File: examples/pytorch/translation/run_translation.py (hunk 1 above)
Comment (SoyGema) [reply examples/pytorch/translation/run_translation.py]: Done I think. thanks

File: examples/pytorch/translation/run_translation.py [hunk 2]
```diff
@@ -378,7 +378,7 @@ def main():
             token=model_args.token,
         )
     # See more about loading any type of standard or custom dataset (from files, python dict, pandas DataFrame, etc) at
-    # https://huggingface.co/docs/datasets/v2.13.1/loading.
+    # https://huggingface.co/docs/datasets/loading.
```
Comment (SoyGema) [inline examples/pytorch/translation/run_translation.py:381]: Remember for future
https://github.com/huggingface/transformers/pull/24594#discussion_r1248095853

**Review by SoyGema — COMMENTED** [review]
point to the official release version

**Pull request merged**